benchmark/buffers/buffer-normalize-encoding.js

```diff
@@ -5,23 +5,18 @@ const common = require('../common.js');
 const bench = common.createBenchmark(main, {
   encoding: [
     'ascii',
-    'ASCII',
     'base64',
     'BASE64',
     'binary',
-    'BINARY',
     'hex',
     'HEX',
     'latin1',
     'LATIN1',
-    'ucs-2',
     'UCS-2',
-    'ucs2',
     'UCS2',
     'utf-16le',
     'UTF-16LE',
     'utf-8',
-    'UTF-8',
     'utf16le',
```

Comment:
We check way to many different types here even after removing a couple of cases. I removed entries in upper and lower case depending on their string length and depending on if at least one variation is still present.

We should likely remove more of these and run the benchmark with some `set` flags in case we change code paths that might change this code.

The normalize code is so fast, it will rarely have any implication in a real application even if the performance difference would be more than 100%.